test/maxRequestsPerSocket.test.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const net = require('net')
+const { test } = require('tap')
+const semver = require('semver')
+const Fastify = require('../fastify')
+
+test('maxRequestsPerSocket on node version greater than 16.10.0', { skip: semver.lt(process.versions.node, '16.10.0') }, t => {
```

Comment:
Github Actions is not updated to node version `16.10.0`.
For those who reviewing this PR should run this test in local environment.

Reply:
It is released.
https://github.com/actions/node-versions/releases/tag/16.10.0-1264673745